monkey/infection_monkey/network/ssh_fingerprinter.py

```diff
@@ -0,0 +1,43 @@
+import re
+from typing import Dict, Tuple, Union
+
+from infection_monkey.i_puppet import FingerprintData, IFingerprinter, PortScanData
+
+SSH_PORT = 22
+SSH_REGEX = r"SSH-\d\.\d-OpenSSH"
+LINUX_DIST_SSH = ["ubuntu", "debian"]
+
+
+class SSHFingerprinter(IFingerprinter):
+    _SCANNED_SERVICE = "SSH"
+
+    def __init__(self):
+        self._banner_regex = re.compile(SSH_REGEX, re.IGNORECASE)
+
+    def get_host_fingerprint(
+        self, host: str, _ping_scan_data, port_scan_data: Dict[int, PortScanData], _options
+    ) -> FingerprintData:
+        os_type = None
+        os_version = None
+        services = {}
+
+        for ps_data in list(port_scan_data.values()):
```

Comment:
`.values()` is already iterable.
```suggestion
        for ps_data in port_scan_data.values():
```